demisto_sdk/commands/common/errors.py

```diff
@@ -553,6 +554,12 @@ def missing_release_notes_for_pack(pack):
                f"update-release-notes -p {pack} -u (major|minor|revision)` to " \
                f"generate release notes according to the new standard."
 
+    @staticmethod
+    @error_code_decorator
+    def missing_release_notes_entry(file_type, pack_name):
+        return f"No release note entry was found for a {file_type.lower()} in the {pack_name} pack. " \
```

Comment:
do we need the pack name here? we will have it in the path

Reply:
the error wording sounds strange if you take out the pack name and it is a duplication I think we can live with.